common/constants.js

```diff
@@ -322,12 +322,14 @@ module.exports = Object.freeze({
   },
   PLATFORM: {
     CF: 'cloudfoundry',
-    K8S: 'kubernetes'
+    K8S: 'kubernetes',
+    SM: 'sapcp'
```

Comment:
I am not sure how that would help? Because consumers could also add their support for their new platforms/managers. So in that case should we drive the entire `PLATFORM` & `PLATFORM_ALIAS_MAPPINGS` objects via config?  
Not exactly because they will anyways add new platform manager and in that PR they might as well update new constants. So i dont think driving it via config will help because what we implement as part of SAPCP Platformmanager might not be really applicable to all customers.

Reply:
Understood. So we accept that this code might be specific to SAP and go ahead.